x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -2196,6 +2207,9 @@ export class RulesClient {
       await pMap(
         response.saved_objects,
         async (rule) => {
```

Comment:
For instance, this async arrow function is a good candidate for extraction into a separate method.

Reply:
Extracted out to `updateRuleAttributesAndParamsInMemory` method